documentation/docs/ui/global_customizations/chat_ui_customizations/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "ChatUI Customizations",
```

Comment:
What do you think about renaming it to `ChatUI`? It's already under `Global Customizations` section